extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -234,6 +234,16 @@
     "actionParameter": "&ck1=$actionParameter1&ck2=$actionParameter2&ck3=$actionParameter3&ck4=$actionParameter4&ck5=$actionParameter5",
     "event": "https://$trackDomain/$trackId/wt?p=432,$contentId,1,_screen_width_x_screen_height_,_screen_color_depth_,1,_timestamp_,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id_&la=_browser_language_&ct=$actionName&ck1=$actionParameter1&ck2=$actionParameter2&ck3=$actionParameter3&ck4=$actionParameter4&ck5=$actionParameter5&pu=_source_url_"
   },
+  "shinystat": {
```

Comment:
apologies for not point it earlier. Same goes here as well (alphabetical).

Reply:
No problems. Fixed.